src/common/types.cpp

```diff
@@ -251,6 +255,8 @@ SQLType TransformStringToSQLType(string str) {
 	} else if (lower_str == "varchar" || lower_str == "bpchar" || lower_str == "text" || lower_str == "string" ||
 	           lower_str == "char") {
 		return SQLType::VARCHAR;
+	} else if (lower_str == "blob") {
```

Comment:
Maybe also add bytea here, as Postgres uses this type, correct?

Reply:
Yeah, I will change it.